CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/editors/LocalDateTimeEditorFactory.java

```diff
@@ -61,6 +61,8 @@ public String getAttributeType() {
         private static final int NUMBER_SPINNER_WIDTH = 55;
         private static final int MILLIS_SPINNER_WIDTH = 60;
         private static final int NANOSECONDS_IN_MILLISECOND = 1000000;
+        
+        private static final String LABEL = "label";
```

Comment:
...especially since it's used all over the place.

Reply:
See response for related comment